File: src/main/java/de/tectoast/emolga/commands/pokemon/FavMonsCommand.java
package de.tectoast.emolga.commands.pokemon;

import de.tectoast.emolga.commands.Command;
import de.tectoast.emolga.commands.CommandCategory;
import de.tectoast.emolga.commands.GuildCommandEvent;

public class FavMonsCommand extends Command {

    public FavMonsCommand() {
        super("favmons", "Erstelle eine Liste deiner Fav Mons", CommandCategory.Pokemon);
        wip();
    }

    @Override
    public void process(GuildCommandEvent e) {

    }
}

File: src/main/kotlin/de/tectoast/emolga/commands/pokemon/FavMonsCommand.kt
package de.tectoast.emolga.commands.pokemon

import de.tectoast.emolga.commands.Command
import de.tectoast.emolga.commands.CommandCategory
import de.tectoast.emolga.commands.GuildCommandEvent

class FavMonsCommand : Command("favmons", "Erstelle eine Liste deiner Fav Mons", CommandCategory.Pokemon) {
    init {
        wip()
    }

    override suspend fun process(e: GuildCommandEvent) {}

    init {
        argumentTemplate = ArgumentManagerTemplate.noArgs()
    }
}